vue-frontend/generate-sitemap.sh

```diff
@@ -0,0 +1,12 @@
+#! /bin/bash
+
+set -e
+
+BASE_URL=$1
+BASE_API_URL=$2
+
+API_URL="$BASE_API_URL/sitemap?baseUrl=$BASE_URL"
+
+$(curl -X GET --header "Accept: */*" $API_URL)
+
+mv ../sitemap/sitemap.xml public/ && rm -r ../sitemap
```

Comment:
Why does this get downloaded in parent directory? Shouldn't it be downloaded in current directory? Also, can we specify download directory? If yes, we can store it in `public/` directory.